tests/fixtures/joint.py

```diff
@@ -43,15 +49,44 @@ class JointFixture:
     mongodb: MongoDbFixture
     rest_client: AsyncTestClient
 
+    def remove_data(self) -> None:
```

Comment:
```suggestion
    def remove_db_data(self) -> None:
```
Would keep naming a bit more explicit as it's not directly obvious what data is being removed at the callsite.